actionview/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   annotated_source returns an empty array instead of nil when the source code
+    can't be found within a TemplateError
```

Comment:
Maybe  "Cast result of `annotated_source_code` to array so `TemplateError`s without a template in the backtrace are surfaced properly by `DebugExceptions`." better explains the situation?